src/psyclone/domain/nemo/transformations/nemo_arrayrange2loop_trans.py

```diff
@@ -139,75 +139,10 @@ def apply(self, node, options=None):
         # Ensure we always use the routine-level symbol table
         symbol_table = node.ancestor(Routine).symbol_table
 
-        # See if there is any configuration information for this array index
-        loop_type_order = Config.get().api_conf("nemo").get_index_order()
-        # TODO: Add tests in get_loop_type_data() to make sure values
-        # are strings that represent an integer or a valid variable
-        # name, e.g. 1a should not be allowed. See issue #1035
-        loop_type_data = Config.get().api_conf("nemo").get_loop_type_data()
-        try:
-            loop_type = loop_type_order[array_index]
-            loop_type_info = loop_type_data[loop_type]
-            lower_bound_info = loop_type_info['start']
-            upper_bound_info = loop_type_info['stop']
-            loop_variable_name = loop_type_info['var']
-        except IndexError:
-            lower_bound_info = None
-            upper_bound_info = None
-            loop_variable_name = symbol_table.next_available_name("idx")
-
-        # Lower bound
-        lower_bound = None
-        if not array_reference.is_lower_bound(array_index):
-            # The range specifies a lower bound so use it
-            lower_bound = node.start.copy()
```

Comment:
This is functionality we need to keep - otherwise we'll always be generating loops over the full range of the array when that might not be what the original code did.

Reply:
Oh I see, thanks.